### PR TITLE
chore: hazop code review changes

### PR DESCRIPTION
**Pull request by SWvheerden**
Description
---
Some changes from the hazop meeting

Motivation and Context
---
Clean secure code

### CODE REVIEW

File: base_layer/core/src/blocks/block.rs [hunk 1]
```diff
@@ -196,10 +194,6 @@ impl BlockBuilder {
 
     /// This function adds the provided transaction kernels to the block WITHOUT updating kernel_mmr_size in the header
     pub fn add_kernels(mut self, mut kernels: Vec<TransactionKernel>) -> Self {
-        for kernel in &kernels {
```
Comment (stringhandler) [inline base_layer/core/src/blocks/block.rs:199]: Is this total fee not used?

**Review by stringhandler — COMMENTED** [review]
Seems ok

File: base_layer/core/src/blocks/block.rs (hunk 1 above)
Comment (SWvheerden) [reply base_layer/core/src/blocks/block.rs:199]: Nope

**Review by brianp — APPROVED** [review]
LGTM utAck

**Pull request merged**